src/lib/components/inventory/list_item_modal.ts

```diff
@@ -468,15 +544,44 @@ export class ListItemModal extends FloatElement {
                                       min="80"
                                       max="120"
                                       step="0.1"
-                                      .value="${this.pricePercentage}"
+                                      .value="${this.pricePercentage.toString()}"
                                       @input="${this.handlePercentageChange}"
                                       class="percentage-slider"
                                   />
+                                  <div class="percentage-assessment-row">
+                                      <div
+                                          class="percentage-assessment-label"
+                                          style="color: ${percentageAssessment.color};"
+                                      >
+                                          <div
+                                              class="percentage-assessment-icon"
+                                              .innerHTML="${percentageAssessment.icon}"
+                                          ></div>
+                                          ${percentageAssessment.label}
+                                      </div>
+                                      <span class="percentage-assessment-value"> ${pricePercentage}% </span>
+                                  </div>
+
                                   <div>
-                                      Percentage of recommended price:
-                                      ${this.recommendedPrice && this.customPrice
-                                          ? Math.round((this.customPrice / this.recommendedPrice) * 100)
-                                          : 100}%
+                                      <label>Visibility</label>
```

Comment:
Toggling visibility to private is pretty rare -- this UI component takes a large amount of space given how infrequent it'll be used.

Perhaps we can right-align it on the same line as the "Visibility" text and make the buttons much smaller?